KIPs/kip-81.md

```diff
@@ -0,0 +1,122 @@
+The newly implemented governance portal includes the following functions: 
+- Ability to propose and vote on a variety of opinions
+- Information about Governance Council members who are working together for the Klaytn ecosystem
+
+The Klaytn governance voting system will be conducted on smart contracts. The contract design is influenced by Compound Finance and OpenZeppelin. The accounts interacting with the contracts can have proposer, voter, or executor roles.
+A proposer submits and initiates governance proposals. The voting timeline starts with the submission of proposals to the voting contract. A proposal describes the changes to be made on the Klaytn blockchain.
+A proposal not only contains textual descriptions of the change, but it may also include  specifications of attached transactions, or actions. The actions are executed on behalf of a voting contract after the proposal passes. The actions extend the impact of proposals from simply gathering opinions to automatically implementing changes on the blockchain.
+A voter casts votes on the proposal during the voting period. A voter can vote for yes, no, or abstain with all the votes assigned to the voter. The list of voters for each proposal is determined at the proposal submission. The number of votes is determined by the amount of staked KLAYs at the start of the voting period.
+An executor triggers the execution of attached transactions in passed proposals. The execution is subdivided into two steps. First, the executor queues the transactions. After a set amount of delay, the executor can send the transactions. Execution delay here gives enough time for the community to recognize the upcoming change and perform a final check about the transaction.
+A proposal is in one of the following states.
+- Pending: Proposer submitted the proposal but voting has not started. The pending state lasts for votingDelay, or 7 days. This state is also called the notice period.
+- Active: Ongoing voting. Voters can cast their votes. The active state lasts for votingPeriod, or 7 days. This state is also called the voting period.
+- Canceled: Proposer has canceled the proposal before execution. Any further interaction is prohibited.
+- Passed: Voting finalized and quorum reached. Executor shall queue the actions before a set timeout of queueTimeout, or 7 days. If the proposal contains no actions, any further interaction is prohibited. Otherwise, the proposal can proceed to Queued and Executed states.
```

Comment:
```suggestion
- Passed: Voting finalized and quorum reached. The executor shall queue the actions before a set timeout of queueTimeout, or 7 days. If the proposal contains no actions, any further interaction is prohibited. Otherwise, the proposal can proceed to Queued and Executed states.
```

1. We need to define `queueTimeout` before this description. What about defining all parameters or variables as a table?
2. If the proposal contains no actions, what is the next state of this proposal?

Reply:
@kjhman21
1. Added tables describing important timings
2. Added a sentence "If the proposal contains no actions, any further interaction is prohibited." in Passed state description.